scripts/run_integration_tests_aarch64.sh

```diff
@@ -197,7 +197,7 @@ sudo bash -c "echo 10 > /sys/kernel/mm/ksm/sleep_millisecs"
 sudo bash -c "echo 1 > /sys/kernel/mm/ksm/run"
 
 export RUST_BACKTRACE=1
-time cargo test --no-default-features --features "integration_tests,kvm" "tests::parallel::$@" -- --skip test_snapshot_restore
+time cargo test --no-default-features --features "integration_tests,kvm" "tests::parallel::$@"
```

Comment:
@rbradford The test case was skipped here. It was not running.

Reply:
Okey dokey. I missed that. What's the path to enabling it with PCI?

Reply:
Indeed it's our next target. But both @MrXinWang and me are lacking some dedicate time to finish a feature like that recently. :(